packages/renderer/src/lib/preferences/PreferencesExtensionList.svelte

```diff
@@ -116,11 +119,11 @@ async function updateExtension(extension: ExtensionInfo, ociUri: string) {
         </div>
       </div>
     </div>
-    <div class="bg-charcoal-600 mt-5 rounded-md p-3">
+    <div class="bg-charcoal-600 mt-5 rounded-md p-3" role="region" aria-label="InstalledExtensions">
       <table class="min-w-full">
         <tbody>
           {#each $extensionInfos as extension}
-            <tr class="border-y border-gray-900">
+            <tr class="border-y border-gray-900" aria-label="{extension.name}">
               <td class="px-6 py-2">
                 <div class="flex items-center">
```

Comment:
It is a pity that we got only `div` elements here. Though, it would be nice, to capture the most important information or a general labels to be added to this vague categorization in HTML, so that we can easily get information:
* default vs. user
* extension version
* if the update is available
* state of the extension

Reply:
not sure how to easily differentiate the user extension from a default extension besides this:
```
                  <div class="ml-4">
                    <div class="flex flex-row">
                      <div class="text-sm text-gray-300">
                        {extension.displayName}
                        {extension.removable ? '(user)' : '(default extension)'}
```
for other stuff, we can check the new aria-labels presence on the page based on the labels I just added